drivers/jason0x43-wemo_switch.groovy

```diff
@@ -37,6 +37,15 @@ metadata {
         command 'subscribe'
         command 'unsubscribe'
         command 'resubscribe'
+        command 'updateAddress',
+            [
+              [name:"IP Address", type: "STRING", description: "New IP address", constraints: []]
+            ]
+        command 'updatePort', 
+            [
+              [name:"TCP Port", type: "STRING", description: "New TCP port", constraints: []]
+            ]
+    }
```

Comment:
This is an extra `}` (syntax error)

Reply:
I don't see it as being extra... one closes the 'definition' section and one closes the 'metadata' section

Reply:
Somehow my local copy didn't match the pasted one... removing the extra one here